string task_id
---
long_term_deployment/TaskGoal goal
